django_tests/all_tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Not yet, and this is something I wanted consultation on.  Travis CI can make a matrix for you and test each combination, if you want.  e.g.:

```
# Tell Travis you want a Python environment to test in
language: python
# List the versions of Python you'd like to test against
python:
  - "2.5"
  - "2.6"
  - "2.7"
# List the versions of Django you'd like to test against
env:
  - DJANGO_VERSION=1.0.4
  - DJANGO_VERSION=1.1.4
  - DJANGO_VERSION=1.2.7
  - DJANGO_VERSION=1.3.1
  - DJANGO_VERSION=1.4
```

(lifted from [here](http://datadesk.latimes.com/posts/2012/06/test-your-django-app-with-travisci/))

But the matrix is trickier- Django 1.7 doesn't work with Python 3.5+, for example.  I could use the matrix and check for compatibility before running the test in the bash part.  Or, we could just stick with the supported Python versions and run all_tests, which will install and test each supported version of Django.